packs/data/spell-effects.db/spell-effect-wild-morph.json

```diff
@@ -135,6 +132,73 @@
                 ],
                 "selector": "fly",
                 "value": 30
+            },
+            {
+                "damageType": "bleed",
+                "dieSize": "d6",
+                "key": "DamageDice",
+                "predicate": [
+                    "wild-morph:wild-claws"
+                ],
+                "selector": "claw-damage",
```

Comment:
Since the strike is granted by the same effect you can skip the predicate and make the selector `{item|_id}-damage`

Reply:
Switched to predicating off slug.